CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ if (NOT XCODE AND NOT MSVC AND NOT CMAKE_BUILD_TYPE)
     set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif()
 
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
```

Comment:
What about `/lib`?

Reply:
bc then the rpath become `$ORIGIN/../lib/libllama.so`, which means i would need to put the binaries into a folder, instead of top level. .... which might actually be better? hmm...